packages/smithy-client/src/parse-utils.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Give an input string, stritctly parses a boolean value.
+ *
+ * @param value The boolean string to parse.
+ * @returns true for "true", false for "false", otherwise an error is thrown.
+ */
+export function parseBoolean(value: "true" | "false"): boolean {
```

Comment:
Prefer arrow function
```suggestion
export const parseBoolean = (value: "true" | "false"): boolean => {
```